antarest/study/storage/rawstudy/model/filesystem/config/files.py

```diff
@@ -171,6 +221,11 @@ def parse_simulation(path: Path) -> Optional["Simulation"]:
             "^([0-9]{8}-[0-9]{4})(eco|adq)-?(.*)", path.stem
         )
         try:
+            if path.suffix == ".zip":
+                zf = ZipFile(path, "r")
+                error = str("checkIntegrity.txt") not in zf.namelist()
```

Comment:
worried about the speed of zf.namelist() for big output on a network disk.
can you test it ?